_locales/en/messages.json

```diff
@@ -158,8 +158,14 @@
     "description":"The button text which says 'Cancel'."
   },
   "saveFilePrompt": {
-    "message": "Do you want to save the file before closing?",
-    "description":"The text which prompts user if they want to save file before closing it."
+    "message": "$filename$ has been modified. Do you want to save it before closing?",
```

Comment:
lgtm :)

Reply:
Is it ok if I do this in a follow up patch? This will probably require waiting on another round of translation as I will need to split the prompt in the messages file into 2 and get a new description translated

Reply:
Sure.
Please file an issue so that we can keep track of it.